code/modules/antagonists/clock_cult/mobs/eminence.dm

```diff
@@ -235,8 +265,48 @@
 		L.visible_message("<span class='warning'>[L] phases out of existance!</span>")
 		var/turf/T = get_turf(pick(GLOB.servant_spawns))
 		try_warp_servant(L, T, FALSE)
+		consume_cogs(E)
 		return TRUE
 	else
 		to_chat(E, "<span class='brass'>You fail to recall [L].</span>")
 		revert_cast(user)
 		return FALSE
+
+//Trigger event
+/obj/effect/proc_holder/spell/targeted/eminence/trigger_event
+	name = "Manipulate Reality"
+	desc = "Manipulate reality causing global events to occur. Costs 5 cogs"
+	action_icon_state = "Geis"
+	charge_max = 3000
+	cog_cost = 5
+
+/obj/effect/proc_holder/spell/targeted/eminence/trigger_event/cast(list/targets, mob/user)
+	var/picked_event = input(user, "Pick an event to run", "Manipulate Reality", null) in list(
+		"Anomaly",
+		"Brand Intelligence",
+		"Camera Failure",
+		"Communications Blackout",
+		"Disease Outbreak",
+		"Electrical Storm",
+		"False Alarm",
+		"Grid Check",
+		"Mass Hallucination",
+		"Processor Overload",
+		"Radiation Storm"
+	)
+	if(!can_cast(user))
+		return
+	if(!picked_event)
+		revert_cast(user)
+		return
+	if(picked_event == "Anomaly")
+		picked_event = pick("Anomaly: Energetic Flux", "Anomaly: Pyroclastic", "Anomaly: Gravitational", "Anomaly: Bluespace")
+	//Reschedule events
+	//Get the picked event
+	for(var/datum/round_event_control/E in SSevents.control)
```

Comment:
better solution here would be to make static associative list instead of looping events but whatever